scIB/metrics.py

```diff
@@ -384,11 +415,20 @@ def max_f1(adata, label_key, cluster_key, label, argmax=False):
         return max_f1
 
     if cluster:
-        opt_louvain(adata_tmp, label_key, cluster_key, function=max_f1, label=label, verbose=False, inplace=True)
+        opt_louvain(
+            adata_tmp,
+            label_key,
+            cluster_key,
+            use_rep=embed,
```

Comment:
The option `use_rep` could lead to different clustering results, which would change the isolated label score.